### PR TITLE
Added announcement channel support

### PR DESCRIPTION
**Pull request by GJSBRT**

### CODE REVIEW

File: core/components/DiscordBot/commands/status.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ export const removeOldEmbed = async (interaction: ChatInputCommandInteraction, t
     const oldMessageId = txAdmin.persistentCache.get('discord:status:messageId');
     if (typeof oldChannelId === 'string' && typeof oldMessageId === 'string') {
         const oldChannel = await interaction.client.channels.fetch(oldChannelId);
-        if (oldChannel?.type === ChannelType.GuildText) {
+        if (oldChannel?.type === ChannelType.GuildText || oldChannel?.type === ChannelType.GuildAnnouncement) {
             await oldChannel.messages.delete(oldMessageId);
         } else {
             throw new Error(`oldChannel is not a guild text channel`);
```
Comment (wowjeeez) [inline core/components/DiscordBot/commands/status.ts]: please highlight the fact that its not an announcement channel either when you throw (like you did with every other error)

Comment (GJSBRT) [reply core/components/DiscordBot/commands/status.ts]: Ah missed that

**Comment by wowjeeez** [comment]
Thanks, looks good otherwise, but the final decision is obviously up to our overlord @tabarra.
Also please make sure that you test it to the fullest extent possible.

**Comment by tabarra** [comment]
Thanks for the PR.

**Pull request merged**